docs/video-player.md

```diff
@@ -0,0 +1,128 @@
+# `<VideoPlayer/>` component for easygoing videos.
```

Comment:
Instead of "easygoing videos", maybe "easy video integration" or "optimized video playback"?

Easygoing videos is technically grammatically fine, but it makes it sound like they're lighthearted films instead of an easy-to-use component.